RHEL/6/input/checks/rsyslog_files_groupownership.xml

```diff
@@ -9,13 +8,25 @@
       </affected>
       <description>All syslog log files should be owned by the
       appropriate group.</description>
+      <reference source="sdw" ref_id="20150408" ref_url="test_attestation" />
     </metadata>
     <criteria>
-      <criterion comment="check if group root owns all syslog log files"
-      test_ref="test_20155" />
+      <criterion comment="check if group root owns all syslog log files" test_ref="test_rsyslog_files_groupownership" />
     </criteria>
   </definition>
-  <ind:unknown_test check="all"
-  comment="use extended content to evaluated this test"
-  id="test_20155" version="1" />
+
+  <unix:file_test check="all" check_existence="all_exist" comment="Files that end in .log in /var/log owned by root" id="test_rsyslog_files_groupownership" version="1">
+    <unix:object object_ref="object_rsyslog_files_groupownership" />
+    <unix:state state_ref="state_rsyslog_files_groupownership" />
+  </unix:file_test>
+
+  <unix:file_object comment="Files that end in .log in /var/log" id="object_rsyslog_files_groupownership" version="2">
+    <unix:path>/var/log</unix:path>
+    <unix:filename operation="pattern match">.*log</unix:filename>
```

Comment:
So what about logs such as ```/var/log/messages``` or ```/var/log/secure``` which do not end in ```.log```? What about logs/dirs that are not managed by rsyslog like ```Xorg.log``` or ```gdm.log``` or ```/var/log/btmp```?